generated/python/grpc-google-cloud-speech-v1beta1/setup.py

```diff
@@ -38,6 +38,6 @@
   install_requires=install_requires,
   license='Apache-2.0',
   packages=find_packages(),
-  namespace_packages=['google.cloud.grpc.speech', 'google.cloud.grpc', 'google.cloud', 'google'],
+  namespace_packages=['google.cloud.grpc.speech', 'google.cloud.grpc', 'google.cloud.proto.speech', 'google.cloud.proto', 'google.cloud', 'google'],
```

Comment:
Same as above -- could you figure out why these `google.cloud.grpc` packages are being included?